src/react-components/src/components/Button/Button.stories.tsx

```diff
@@ -111,3 +113,25 @@ export const linkLikeButtonIconLeft = () => (
         <span>button content</span>
     </Button>
 );
+
+buttonFilledIcon.story = {
```

Comment:
Can we maybe keep these alongside the story they add documentation for? i.e.

```
export const buttonFilledIcon = () => (
...
);

buttonFilledIcon.story = {
...
};
```

Reply:
I updated to move the Figma links closer to their related stories!